packages/server/src/admin/admin-entities.ts

```diff
@@ -0,0 +1,19 @@
+import { AdminEntity } from 'nestjs-admin';
+import { CourseModel } from 'course/course.entity';
+import { QueueModel } from 'queue/queue.entity';
+import { UserModel } from 'profile/user.entity';
+
+export class CourseAdmin extends AdminEntity {
```

Comment:
These classes basically tell nestjs-admin how to display the entity. They are optional, as you can also just register the plain typeorm model, but it's nice to show more than just the id in the list. 
We can further customize which fields are shown and which can be searched: https://nestjs-admin.com/docs/admin-entity